trollflow2/plugins/__init__.py

```diff
@@ -257,11 +258,13 @@ def save_datasets(job):
     This is useful when other processes are waiting for the file to be present
     to start their work, but would crash on incomplete files.
 
-    If, in addition, the ``use_tmp_dir`` option is provided in the product
-    list, then the temporary file will be created in this directory without
+    If the ``staging_zone`` option is provided in the product
+    list, then the file will be created in this directory first, without
```

Comment:
without changing the filename, you mean keeping either original or tmp file depending on `use_tmp_file`? (seems to be so if I understand the unit tests). Maybe this should be clarified?

Reply:
Your interpretation is correct.  I clarified the documentation (see update).